Makefile.am

```diff
@@ -0,0 +1,144 @@
+SUBDIRS = cppForSwig
+PREFIX = @prefix@
+
+all:
+	$(MAKE) -C cppForSwig
+
+clean-local:
+	rm -f CppBlockUtils.py
+	rm -f qrc_img_resources.py
+	rm -f _CppBlockUtils.so
+	rm -f cppForSwig/cryptopp/a.out
+	rm -f *.pyc BitTornado/*.pyc bitcoinrpc_jsonrpc/*.pyc ui/*.pyc
+	rm -f armoryengine/*.pyc dialogs/*.pyc BitTornado/BT1/*.pyc
+	rm -f pytest/*.pyc txjsonrpc/*.pyc jsonrpc/*.pyc txjsonrpc/web/*.pyc
+
+install-data-local:
+	$(MKDIR_P) $(DESTDIR)$(datarootdir)/applications
+	$(MKDIR_P) $(DESTDIR)$(datarootdir)/armory/img
+	rsync -rupE --exclude="img/.DS_Store" img $(DESTDIR)$(datarootdir)/armory/
+	$(SED) "s:python /usr:python $(PREFIX):g" < dpkgfiles/armory.desktop > $(DESTDIR)$(datarootdir)/applications/armory.desktop
+	$(SED) "s:python /usr:python $(PREFIX):g" < dpkgfiles/armoryoffline.desktop > $(DESTDIR)$(datarootdir)/applications/armoryoffline.desktop
+	$(SED) "s:python /usr:python $(PREFIX):g" < dpkgfiles/armorytestnet.desktop > $(DESTDIR)$(datarootdir)/applications/armorytestnet.desktop
+
+install-exec-local:
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/extras
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/bitcoinrpc_jsonrpc
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/txjsonrpc
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/txjsonrpc/web
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/ui
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/pytest
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/BitTornado/BT1
+	$(MKDIR_P) $(DESTDIR)$(libdir)/armory/urllib3
+	$(MKDIR_P) $(DESTDIR)$(bindir)
+	$(SED) "s: /usr/lib: $(libdir):g" < dpkgfiles/armory > $(DESTDIR)$(bindir)/armory
+	chmod +x $(DESTDIR)$(bindir)/armory
+	cp *.py *.so README.md $(DESTDIR)$(libdir)/armory/
+	rsync -rupE armoryengine $(DESTDIR)$(libdir)/armory/
+	cp extras/*.py $(DESTDIR)$(libdir)/armory/extras
+	cp bitcoinrpc_jsonrpc/*.py $(DESTDIR)$(libdir)/armory/bitcoinrpc_jsonrpc
+	cp -R txjsonrpc/*.py $(DESTDIR)$(libdir)/armory/txjsonrpc
+	cp -R txjsonrpc/web/*.py $(DESTDIR)$(libdir)/armory/txjsonrpc/web
+	cp ui/*.py $(DESTDIR)$(libdir)/armory/ui
+	cp pytest/*.py $(DESTDIR)$(libdir)/armory/pytest
+	cp -R urllib3/* $(DESTDIR)$(libdir)/armory/urllib3
+	cp BitTornado/*.py $(DESTDIR)$(libdir)/armory/BitTornado
+	cp BitTornado/BT1/*.py $(DESTDIR)$(libdir)/armory/BitTornado/BT1
+	cp default_bootstrap.torrent $(DESTDIR)$(libdir)/armory
+
+all-test-tools: all
+	$(MAKE) -C cppForSwig/gtest
+
+gtest: all-test-tools
+	(cd cppForSwig/gtest && ./CppBlockUtilsTests)
+
+pytest: all
+	python -m unittest discover
+
+test: gtest pytest
+
+mo: messages
```

Comment:
Much cleaner to just integrate the po/mo targets here so they can be used as dependencies